### ISSUE TITLE
Eden tokens

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1093]
should have ∞ amount of tokens if you 100%-ed the game (hope this is not a duplicate)

**Comment by Lonrite** [comment]
forgot to mention but ∞ tokens also looks more visually appealing than some random number. considering how he got improved hairstyles just recently this change would work wonders as well 
                       
   ( eden also contains infinite beings inside of himself so that part goes in line with that perfectly )

**Comment by ghost** [comment]
Agreed with this, running out of tokens while streaking Eden is very annoying. And if it is only on Dead God files then it also can't be abused.
